README.md

```diff
@@ -252,6 +252,14 @@ Mbed TLS can be ported to many different architectures, OS's and platforms. Befo
 -   [What external dependencies does Mbed TLS rely on?](https://tls.mbed.org/kb/development/what-external-dependencies-does-mbedtls-rely-on)
 -   [How do I configure Mbed TLS](https://tls.mbed.org/kb/compiling-and-building/how-do-i-configure-mbedtls)
 
+Mbed TLS is mostly written in portable C99; however, it has a few platform requirements that go beyond the standard, but are met by most modern architectures:
+
+- Bytes must be 8 bits.
+- `all-bits-zero` must be a valid representation of a null pointer.
```

Comment:
Nit: This seems like there's some constant `all-bits-zero` that must be defined, because of the backticks.
```suggestion
- All-bits-zero must be a valid representation of a null pointer.
```